src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,194 @@
+    const configStrColored = util.inspect(config, false, undefined, true)
+    const configStrUncolored = JSON.stringify(config, undefined, 2)
+
+    // Print config
+    this.context.stdout.write(`\n${configStrColored}\n`)
+    if (this.isDryRun) {
+      this.context.stdout.write('\n🚫 Configuration not sent because the command was ran as a dry run.\n')
+
+      return 0
+    }
+
+    // Write config to file
+    const folderPath = path.join(process.cwd(), '.lambda-flare-output')
+    if (!fs.existsSync(folderPath)) {
+      fs.mkdirSync(folderPath)
+    }
+    const filePath = path.join(folderPath, 'function_config.json')
+    fs.writeFileSync(filePath, configStrUncolored)
+
+    // Zip folder
+    const zipPath = path.join(folderPath, 'lambda-flare-output.zip')
+    const zipFolder = async (command: Command) => {
+      try {
+        const data = await fs.promises.readFile(filePath, 'utf8')
+        const zip = new JSZip()
+        zip.file('function_config.json', data)
+        const content = await zip.generateAsync({type: 'nodebuffer'})
+        await fs.promises.writeFile(zipPath, content)
+      } catch (error) {
+        command.context.stderr.write(renderError('Error reading file or writing zip file. Please try again.'))
+
+        return true
+      }
+
+      return false
+    }
+    if (await zipFolder(this)) {
+      return 1
+    }
+
+    // Send to Datadog
+    this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
+    const form = new FormData()
+    form.append('case_id', this.caseId)
+    form.append('flare_file', fs.createReadStream(zipPath))
+    form.append('operator_version', 7)
+    form.append('email', this.email)
+    const requestConfig = {
+      headers: {
+        ...form.getHeaders(),
+        'DD-API-KEY': this.apiKey,
+      },
+    }
+    await axios
+      .post('https://datad0g.com/api/ui/support/serverless/flare', form, requestConfig)
+      .then(() => {
+        this.context.stdout.write('\n✅ Successfully sent function config to Datadog Support!\n')
+      })
+      .catch((err) => {
+        this.context.stderr.write(
+          renderError('Failed to send function config to Datadog Support. Is your email and case ID correct?')
+        )
+        this.context.stderr.write(renderError(err))
+      })
+
+    // Remove file
+    const deleteFolderContents = (dir: string) => {
+      if (fs.existsSync(dir)) {
+        fs.readdirSync(dir).forEach((file) => {
+          const currentPath = path.join(dir, file)
+          if (fs.lstatSync(currentPath).isDirectory()) {
+            deleteFolderContents(currentPath)
+          } else {
+            fs.unlinkSync(currentPath)
+          }
+        })
+      }
+    }
+    deleteFolderContents(folderPath)
+    try {
+      fs.rmdirSync(folderPath)
+    } catch (e) {
+      this.context.stderr.write(renderSoftWarning('Failed to delete log files located at .lambda-flare-output'))
```

Comment:
`renderSoftWarning` or `renderError`?

Reply:
Soft warning is perfect. Also what do you mean by `log files`? Wouldn't it be `Flare files?` Also, if you are going to give them the path, dont be arbritrary and specify the full path. The folder instance must have a `toString` that gives you the full path.